proto/dwtools/abase/l0.test/Long.test.s

```diff
@@ -11899,6 +11951,50 @@ function longAnyAreRepeated( test )
   var expected = true;
   test.identical( got, expected );
 
+  /* with evaluator, equalizer */
+
+  test.case = 'empty array with evaluator';
+  var evaluator = ( e ) => e;
+  var src = [];
```

Comment:
Long type is a complex type. So, add same test cases for unrolls. See [all Long types](https://github.com/dmvict/wTools/blob/master/doc/ukr/concept/TypeIndexed.md)